src/core/transforms/WebRTC.tsx

```diff
@@ -170,23 +170,34 @@ export const RoomParticipant = {
               </div>
             )}
           </div>
-          <video
-            ref={ref}
-            autoPlay={true}
-            muted={muteAudio}
-            disablePictureInPicture={true}
-            playsInline={true}
+          <div
             style={{
-              left: '50%',
-              top: '50%',
               position: 'relative',
-              transform: 'translate3d(-50%, -50%, 0)',
+              display: 'flex',
               height: '100%',
-              opacity: hasVideo ? '1' : '0',
-              objectFit: source?.props.type === 'screen' ? 'contain' : 'cover',
-              background: 'rgba(0,0,0,0.6)',
+              width: '100%',
+              transform: source?.props?.mirrored && 'scaleX(-1)',
```

Comment:
I think that generally a better pattern for having optional properties like this would be

```
...(Boolean(source?.props?.mirrored) &&
      { transform: 'scaleX(-1)' }
    ),
```

As this prevents `transform` from being set to falsey values such as `null` or `0`. I wouldn't expect this to be a problem, but I think it's just a better pattern.

Reply:
Good call!